x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/models/graph/if_vertex.js

```diff
@@ -37,21 +37,37 @@ export class IfVertex extends Vertex {
   }
 
   get trueEdge() {
-    return this.outgoingEdges.find(e => e.when === true);
+    return this.outgoingEdges.find(e => e.when);
```

Comment:
@ycombinator I removed the explicit `true` check to make this uniform with the `true` checks I've added - alternatively I could change this back and make it explicit for the other comparisons in the file as well. WDYT?

Reply:
I personally like the explicit check but I'm good with this change too, so up to you.

Reply:
I prefer them as well, I'll change change it.